packages/hanging-garden/src/renderHooks/useScrolling.tsx

```diff
@@ -55,11 +55,13 @@ const useScrolling = <T extends HangingGardenColumnIndex>(
         return false;
       }
 
-      scrollLeft.current =
+      scrollLeft.current = container.current.scrollLeft = Math.max(
```

Comment:
either assign value to variable or create a function for calculation or a comment that explains the calculation of offset.